### PR TITLE
🏇 Personal Profile: Adding Deletions Tests

### LINKED ISSUE #69: 🏇 Personal Profile: Adding Deletion Tests
Including deletion tests in tests.py of Personal Profile:

- [x] Test if the profile gets deleted when we use `.delete()`
- [x] Test if the profile gets deleted when its user is deleted

### PR DESCRIPTION
**Pull request by ShirleyFichman**
In this PR there were added deletion tests for the personal profile model.
Close #69.

### CODE REVIEW

**Review by DeanBiton — APPROVED** [review]
LGTM

File: personal_profile/tests.py [hunk 1]
```diff
@@ -28,16 +30,36 @@ def profile_1(db, user_1):
...
+        test_profile.delete()
+        assert test_profile not in PersonalProfile.objects.all()
+
+
+@pytest.mark.django_db
+class TestProfileUserRelation:
+    # Testing personal profile model and user relation
+
+    def test_profile_is_deleted_when_user_is_deleted(self, profile_1, user_1):
+        profile_1.save()
```
Comment (paOmer) [inline personal_profile/tests.py:63]: This line in unnecessary since you've saved the profile in the fixture setup at line 33.

Comment (Yarboa) [reply personal_profile/tests.py:63]: please rebase and push

Comment (ShirleyFichman) [reply personal_profile/tests.py]: done

**Review by paOmer — CHANGES_REQUESTED** [review]
This changes should make the tests pass successfully

**Review by paOmer — CHANGES_REQUESTED** [review]
This changes should fix flake8 requirements

File: personal_profile/tests.py [hunk 2]
```diff
@@ -36,25 +40,27 @@ class TestProfileModel:
...
+        assert test_profile.birth_date == profile_1.birth_date
+        assert test_profile.profile_pic == profile_1.profile_pic
+        assert test_profile.resume == profile_1.resume
 
-    def test_profile_is_deleted(self, profile_1):
-        profile_1.save()
-        assert profile_1 in PersonalProfile.objects.all()
-        profile_1.delete()
-        assert profile_1 not in PersonalProfile.objects.all()
+    def test_profile_is_deleted(self, user_1):
```
[the remark this answers is not in the document]
Comment (ShirleyFichman) [reply personal_profile/tests.py]: done

File: personal_profile/tests.py [hunk 3]
```diff
@@ -36,25 +40,27 @@ class TestProfileModel:
...
 
 @pytest.mark.django_db
 class TestProfileUserRelation:
     # Testing personal profile model and user relation
 
-    def test_profile_is_deleted_when_user_is_deleted(self, profile_1, user_1):
-        profile_1.save()
-        user_1.delete()
-        assert profile_1 not in PersonalProfile.objects.all()
+    def test_profile_is_deleted_when_user_is_deleted(self, user_1):
```
[the remark this answers is not in the document]
Comment (ShirleyFichman) [reply personal_profile/tests.py]: done

File: personal_profile/tests.py [hunk 4]
```diff
@@ -1,6 +1,7 @@
 import pytest
 import django.contrib.auth
```
[the remark this answers is not in the document]
Comment (ShirleyFichman) [reply personal_profile/tests.py]: done

File: personal_profile/tests.py [hunk 5]
```diff
@@ -36,25 +40,27 @@ class TestProfileModel:
...
 class TestProfileUserRelation:
     # Testing personal profile model and user relation
 
-    def test_profile_is_deleted_when_user_is_deleted(self, profile_1, user_1):
-        profile_1.save()
-        user_1.delete()
-        assert profile_1 not in PersonalProfile.objects.all()
+    def test_profile_is_deleted_when_user_is_deleted(self, user_1):
+        test_profile = PersonalProfile.objects.filter(user=user_1).first()
+        test_user = User.objects.filter(password='userpassword').first()
```
[the remark this answers is not in the document]
Comment (ShirleyFichman) [reply personal_profile/tests.py]: I tried it before but I will try again, thank you for all the suggestions!

**Review by paOmer — APPROVED** [review]
LGTM!

**Review by Yarboa — APPROVED** [review]
LGTM

**Pull request merged**